### ISSUE TITLE
Dim system audio when engaged

### ISSUE THREAD

**Issue by at-avacron** [yulrizka/osx-push-to-talk#16]
Hi, this is a feature request from a user. I'm an audio engineer, and I have remote clients use OSX Push to Talk so that they can leave the audio input open to a high quality audio stream, which unlike Meet or Zoom does not have an algorithm to cancel the audio that I am sending to them. In order to avoid feedback, the client has to use headphones, which is inconvenient on several levels.

The solution to the resultant feedback loop is to have their system audio "dim," i.e. reduce by a user-adjustable amount, when their push-to-talk is engaged. In the pro audio world, every console has a Talkback button, and every Talkback button has an associated circuit to dim the control room speakers. This would accomplish the same thing for anyone, on any laptop, anywhere. I have not found any software product (paid or free) that can accomplish this simple task.

**Comment by yulrizka** [comment]
I'm not fully sure I get the request, do you mean instead of mute, we just reduce the volume to an adjustable setting?

**Comment by at-avacron** [comment]
Yes, an output volume reduction. It would be a menu item, a simple slider. If it were an always-active option, people who didn't need the feature could just leave "Dim" at full (Dim is actually the common term FWIW). If it were an option you had to enable, I suppose it could be greyed out for people who don't need it.

I'm not sure how many engineers are using this, but that one feature would immediately put this in a league above *many* other options that are supposedly tailored for us. Cheers.
